runner/app/live/streamer/process_guardian.py

```diff
@@ -122,6 +127,9 @@ def _current_state(self) -> str:
         input = self.status.input_status
         last_input_time = input.last_input_time or 0
         time_since_last_input = current_time - last_input_time
+        # Stream is done and 10s grace period reached
+        if not self.stream_alive and time_since_last_input > 10:
```

Comment:
The grace period was mostly to give time to the `_do_stop` to run (sometimes closing trickle stream can take a while for example). Since we are calling `stop_stream` in the end of `_do_stop`, we can either remove the grace period or make it much shorter, maybe 1-5s.

Reply:
ok, changed to 5s